utils/log/logger.go

```diff
@@ -18,9 +18,9 @@ type LogFormat string
 // Used for coloring sections of the log message. For example log.Format.Path("...")
 var Format LogFormat
 
-// Determines whether the terminal is available. This variable should not be accessed directly,
-// but through the 'isTerminalMode' function.
-var terminalMode *bool
+// Determines whether the terminal is available. This variable should not be accessed directly (besides  tests),,
+// but through the 'isTerminal' function.
+var TerminalMode *bool
```

Comment:
What's the reason for making this variable public?

Reply:
We need to be able to change the terminal environment because tests may run in a non-terminal environment.